src/client/containers/fetchMethod/fetch.js

```diff
@@ -24,6 +24,9 @@ export async function fetchFromDb(endpoint, fetchMethod, postBody = {}) {
   if (!response.ok) {
     throw new ApiError(response.statusText, response.status);
   }
-  const dbData = await response.json();
-  return dbData;
+  if (fetchMethod === 'get') {
+    const dbData = await response.json();
+    return dbData;
+  }
+  return response;
```

Comment:
Hmm, this probably makes sense for now but in general also other requests may return data and sometimes it is interesting to access the response object in GET requests... You're sure this doesn't break any existing code?

Reply:
I could just return the response for all the methods if someone wants to access any other data of the response object. I tried it with get, post, put and delete and it seems to work fine, cant said I tried it with all the endpoints still.